framework/platform/plugins/plugin.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2020-2021, Arm Limited and Contributors
+/* Copyright (c) 2020-2024, Arm Limited and Contributors
```

Comment:
Nothing changed here but the copyright year?

Reply:
Fixed